packit_service/models.py

```diff
@@ -1084,88 +1096,88 @@ def __repr__(self):
         )
 
 
-AbstractTriggerDbType = Union[
+AbstractProjectEventDbType = Union[
     PullRequestModel,
     ProjectReleaseModel,
     GitBranchModel,
     IssueModel,
 ]
 
-MODEL_FOR_TRIGGER: Dict[JobTriggerModelType, Type[AbstractTriggerDbType]] = {
-    JobTriggerModelType.pull_request: PullRequestModel,
-    JobTriggerModelType.branch_push: GitBranchModel,
-    JobTriggerModelType.release: ProjectReleaseModel,
-    JobTriggerModelType.issue: IssueModel,
+MODEL_FOR_TRIGGER: Dict[ProjectEventModelType, Type[AbstractProjectEventDbType]] = {
+    ProjectEventModelType.pull_request: PullRequestModel,
+    ProjectEventModelType.branch_push: GitBranchModel,
+    ProjectEventModelType.release: ProjectReleaseModel,
+    ProjectEventModelType.issue: IssueModel,
 }
 
 
-class JobTriggerModel(Base):
+class ProjectEventModel(Base):
     """
     Model representing a trigger of some packit task.
 
     It connects PipelineModel (and built/test models via that model)
-    with models like PullRequestModel, GitBranchModel or ProjectReleaseModel.
+    with models like IssueModel, PullRequestModel, GitBranchModel or ProjectReleaseModel.
 
     * It contains type and id of the other database_model.
       * We know table and id that we need to find in that table.
-    * Each PipelineModel has to be connected to exactly one JobTriggerModel.
-    * There can be multiple PipelineModels for one JobTriggerModel.
-      (e.g. For each push to PR, there will be new PipelineModel, but same JobTriggerModel.)
+    * Each PipelineModel has to be connected to exactly one ProjectEventModel.
+    * There can be multiple PipelineModels for one ProjectEventModel.
+      (e.g. For each push to PR, there will be new PipelineModel, but same ProjectEventModel.)
     """
 
-    __tablename__ = "job_triggers"
+    __tablename__ = "project_events"
```

Comment:
This will require manual intervention during the migration to not loose data... (Alembic will remove the old one and create a new one...)

Reply:
You can also leave the table as is and rename it manually as a follow-up.

Reply:
Aha, ok. You've already handled the renaming..;)